BeatTogether.DedicatedServer.Kernel/Managers/LobbyManager.cs

```diff
@@ -88,10 +90,42 @@ private async void UpdateLoop(CancellationToken cancellationToken)
         public void Update()
         {
             if (_instance.State != MultiplayerGameState.Lobby)
-                return;
+            {
+
+                //Sends players stuck in the lobby to spectate the ongoing game, prevents a rare quest issue with loss of tracking causing the game to pause on map start
```

Comment:
Sends players stuck in the lobby to spectate the ongoing game, prevents a rare quest issue with loss of tracking causing the game to pause on map start.

Reply:
BUGFIX